src/site/content/en/blog/streams/index.md

```diff
@@ -0,0 +1,966 @@
+contain any of the following methods:
+
+### The `transformer`
+
+- `start(controller)`: This method is called immediately when the object is constructed. Typically
+  this is used to enqueue prefix chunks, using `controller.enqueue()`. Those chunks will be read
+  from the readable side but do not depend on any writes to the writable side. If this initial
+  process is asynchronous, for example because it takes some effort to acquire the prefix chunks,
+  the function can return a promise to signal success or failure; a rejected promise will error the
+  stream. Any thrown exceptions will be re-thrown by the `TransformStream()` constructor.
+- `transform(chunk, controller)`: This method is called when a new chunk originally written to the
+  writable side is ready to be transformed. The stream implementation guarantees that this function
+  will be called only after previous transforms have succeeded, and never before `start()` has
+  completed or after `flush()` has been called. This function performs the actual transformation
+  work of the transform stream. It can enqueue the results using `controller.enqueue()`. This
+  permits a single chunk written to the writable side to result in zero or multiple chunks on the
+  readable side, depending on how many times `controller.enqueue()` is called. If the process of
+  transforming is asynchronous, this function can return a promise to signal success or failure of
+  the transformation. A rejected promise will error both the readable and writable sides of the
+  transform stream. If no `transform()` method is supplied, the identity transform is used, which
+  enqueues chunks unchanged from the writable side to the readable side.
+- `flush(controller)`: This method is called after all chunks written to the writable side have been
+  transformed by successfully passing through `transform()`, and the writable side is about to be
+  closed. Typically this is used to enqueue suffix chunks to the readable side, before that too
+  becomes closed. If the flushing process is asynchronous, the function can return a promise to
+  signal success or failure; the result will be communicated to the caller of
+  `stream.writable.write()`. Additionally, a rejected promise will error both the readable and
+  writable sides of the stream. Throwing an exception is treated the same as returning a rejected
+  promise.
+
+### The `writableStrategy` and `readableStrategy` queueing strategies
+
+The second and third optional argument to the `TransformStream()` constructor takes optional
+`writableStrategy` and `readableStrategy` queueing strategies. They are defined as outlined in the
+[readable](/streams/#the-queuingstrategy) and the [writable](/streams/#the-queuingstrategy-2) stream
+sections respectively.
+
+### Transform stream code sample
+
+The following code sample shows a simple transform stream in action.
+
+```js
+const transformStream = new TransformStream({
```

Comment:
```suggestion
const textEncoderStream = new TransformStream({
```

Also, maybe mention this exists in the platform already?

Reply:
Also mentioned that this exists now.